mmcv/ops/csrc/pytorch/ball_query.cpp

```diff
@@ -0,0 +1,36 @@
+// Modified from
+// https://github.com/sshaoshuai/Pointnet2.PyTorch/tree/master/pointnet2/src/ball_query.cpp
+
+#include "pytorch_cpp_helper.hpp"
+
+#ifdef MMCV_WITH_CUDA
+void ball_query_kernel_launcher(int b, int n, int m, float min_radius,
+                                float max_radius, int nsample, const float *xyz,
+                                const float *new_xyz, int *idx);
+#endif
+
+int ball_query_wrapper(int b, int n, int m, float min_radius, float max_radius,
+                       int nsample, at::Tensor new_xyz_tensor,
+                       at::Tensor xyz_tensor, at::Tensor idx_tensor);
+
+int ball_query_wrapper(int b, int n, int m, float min_radius, float max_radius,
```

Comment:
should we rename the `ball_query_wrapper` to `ball_query_forward` which is the convention of mmcv?